fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -206,6 +208,22 @@ private void createOutputFragment(PlanFragment inputFragment, ExecPlan execPlan,
         execPlan.getFragments().add(exchangeFragment);
     }
 
+    void useQueryCache(ExecPlan execPlan) {
```

Comment:
would better split this method to two method.  one method check whether should use the query cache, one method normalize the fragment.